specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+            "description": "Error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Snapshot_Get": {
+            "$ref": "./examples/Snapshot_Get.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.AppComplianceAutomation/reports/{reportName}/snapshots/{snapshotName}/download": {
+      "post": {
+        "tags": [
+          "AppComplianceAutomation"
+        ],
+        "description": "Download compliance needs from snapshot, like: Compliance Report, Resource List.",
+        "operationId": "Snapshot_Download",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ReportNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SnapshotNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ReportCreatorTenantIdParameter"
+          },
+          {
+            "$ref": "#/parameters/DownloadTypeParameter"
+          },
+          {
+            "$ref": "#/parameters/OfferGuidParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/DownloadResponse"
+            }
+          },
+          "202": {
+            "description": "Accepted. The response indicates the exiting AppComplianceAutomation report is now updating and contains a Location header to query the operation result."
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Snapshot_Download_ComplianceReport": {
+            "$ref": "./examples/Snapshot_ComplianceReport_Download.json"
+          },
+          "Snapshot_Download_CompliancePdfReport": {
+            "$ref": "./examples/Snapshot_CompliancePdfReport_Download.json"
+          },
+          "Snapshot_Download_ComplianceDetailedPdfReport": {
+            "$ref": "./examples/Snapshot_ComplianceDetailedPdfReport_Download.json"
+          },
+          "Snapshot_Download_ResourceList": {
+            "$ref": "./examples/Snapshot_ResourceList_Download.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OverviewStatus": {
+      "description": "The overview of the compliance result for one report.",
+      "type": "object",
+      "properties": {
+        "passed": {
```

Comment:
>passed [](http://example.com/codeflow?start=9&length=6)

nit : consider being more descriptive

passedCount , failedCount, etc

Reply:
@raosuhas Thanks for your comments. We can think about how to migrate this variable name.

Reply:
Hi @raosuhas ~
Fixed! Please review [here](https://github.com/Azure/azure-rest-api-specs/pull/21152/files#diff-16cc0066db31d47ee8b4f0e9385ece034881677d74a14b4d48790444b24bda4dL519).